source/language/types.rst

```diff
@@ -268,6 +268,72 @@ are all of type ``float[64]``.
 
 Note that `e` is a valid identifier. `e/E` are also used in scientific notation where appropriate.
 
+Mathematical functions available for constant initialization
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In addition to simple arithmetic functions used in expressions initializing constants,
+OpenQASM 3 offers the following built-in mathematical operators followed by
+their argument expression in parentheses:
+
+.. container::
+   :name: tab:built-in-math
+
+   .. table:: Built-in mathematical functions in OpenQASM3.
+
+      +----------+--------------------------------+---------------------+
+      | Function | [Input Range/Type] , [...]     | [Output Range/Type] |
+      +==========+================================+=====================+
+      | arccos   | [-1, 1]                        | [0, pi rad]         |
+      +----------+--------------------------------+---------------------+
+      | arcsin   | [-1, 1]                        | [-pi/2, pi/2 rad]   |
+      +----------+--------------------------------+---------------------+
+      | arctan   | [-∞, Real, ∞]                  | [-pi/2, pi/2 rad]   |
+      +----------+--------------------------------+---------------------+
+      | ceiling  | [float], [float]               | [float]             |
+      +----------+--------------------------------+---------------------+
+      | cos      | [float]                        | [float]             |
+      |          |                                |                     |
+      |          | [angle]                        | [float]             |
+      +----------+--------------------------------+---------------------+
+      | div      | ??                             | ??                  |
```

Comment:
Let's just remove this from the list until there's a compelling reason to add it - if we can't even define it (e.g. how does it differ from the `/` operator?), it's probably an indication we don't want it.